tasks/main.yml

```diff
@@ -2,8 +2,8 @@
 - name: Install collectd.
   yum: name={{ item }} state=latest
   with_items:
-    - collectd
-    - collectd-netlink
+    - collectd{{ aerospike_collectd_package_version | default('') }}
+    - collectd-netlink{{ aerospike_collectd_package_version | default('') }}
```

Comment:
Forgot, can we easily have a concat of `-` and param? The `-` in the config is a bit weird if we don't look at how it is setup here.